src/api.js

```diff
@@ -9,4 +9,10 @@ export const getArticles = () => {
     }).catch((err) => {
         console.log(err)
     })
+}
+
+export const getArticleById = (article_id) => {
+    return api.get(`/articles/${article_id}`).then((response) => {
+        return response.data
+    })
```

Comment:
Nice, but there should be a key of .article on data here which should give us the article we have requested for. This may be due to the backend not being setup correctly to send this on the correct key.